lms/djangoapps/support/views/contact_us.py

```diff
@@ -5,12 +5,17 @@
 
 from edxmako.shortcuts import render_to_response
 
+from student.models import CourseEnrollment
+
 
-#TODO https://openedx.atlassian.net/browse/LEARNER-2296
 class ContactUsView(View):
     """
     View for viewing and submitting contact us form.
     """
 
     def get(self, request):
-        return render_to_response("support/contact_us.html")
+        context = {}
+        if request.user.is_authenticated():
+            context['user_enrollments'] = CourseEnrollment.enrollments_for_user(request.user)
```

Comment:
Should we filter out enrollments that's no longer `active`?

Reply:
@schenedx I tested this on stage. I can submit request for archived courses as well. Let me know if we really want this change ?

Reply:
Looks like the API call `enrollments_for_user` already filter out `active=0` https://github.com/edx/edx-platform/blob/master/common/djangoapps/student/models.py#L1479
We are good then.